csharp/test/Drivers/Apache/Spark/SparkTestEnvironment.cs

```diff
@@ -139,6 +139,13 @@ public override string GetInsertStatement(string tableName, string columnName, s
         public override SampleDataBuilder GetSampleDataBuilder()
         {
             SampleDataBuilder sampleDataBuilder = new();
+            Type floatNetType = ServerType == SparkServerType.Databricks ? typeof(float) : typeof(double);
+            Type floatArrowType = ServerType == SparkServerType.Databricks ? typeof(FloatType) : typeof(DoubleType);
+            object floatValue;
```

Comment:
In principle, it probably should be float for both. I imagine the reason we get doubles via Thrift is that TColumn doesn't define a TFloatColumn, only a TDoubleColumn. But if we know from the schema that it should be a float then we should probably be doing a conversion in the driver.